css/popup.css

```diff
@@ -0,0 +1,160 @@
+.project-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  background: rgba(0, 0, 0, 0.8);
+  fill: blur(40px);
+}
+
+.popup-container {
+  background-color: var(--white);
+  padding: 1.5rem;
+  position: relative;
+}
+
+.popup-container h2 {
+  font-size: 2rem;
+  line-height: 2.75rem;
+  font-style: normal;
+  color: #172b4d;
+}
+
+.tech-stack {
+  display: flex;
+  margin-bottom: 1.325rem;
+}
```

Comment:
- [ ] Please add flex-wrap: wrap; and gap: 10px; to make the design looks like the Figma design.